metadata.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "puppetlabs-secure_env_vars",
+  "version": "0.1.0",
+  "author": "Puppet",
+  "summary": "Run a command or script with sensitive environment variables.",
+  "license": "Apache-2.0",
+  "source": "https://github.com/puppetlabs/puppetlabs-secure_env_vars",
+  "dependencies": [
+    {
+      "name": "puppetlabs-stdlib",
+      "version_requirement": ">= 6.0.0"
```

Comment:
This seems high, particularly since many modules have a maximum version of 5.x - I think we can go back to maybe 4.24, which is https://github.com/puppetlabs/puppetlabs-stdlib/commit/21e7ce8fc3cdacab50b71d1973d7c80ed0543fd3#diff-a03c09e759361331222b5070179514a282f9b3a949b91f133f364b309d20a189